pages/newsroom/styles.module.scss

```diff
@@ -0,0 +1,16 @@
+.newsroomContainer {
+  background-image: url(../../img/backgrounds/mesh.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100vh;
```

Comment:
Did someone give you a comp to do this? Odd choice to make it this way...
Also, doesn't `100vh` interfere with the header?